src/02-crm/tags.apib

```diff
@@ -0,0 +1,29 @@
+## Tags [/tags]
+
+### tags.list [GET /tags.list]
+
+Get a list of tags.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + tag
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+            + Default
+                + (object)
+                    + field: `tag`
+                    + order: `asc`
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + tag: `campaign` (string)
```

Comment:
I think we wanted an array of `string`s before? But I think this allows us to add extra fields later. I do however have a remark on the key name. I think we should consider the object being the `Tag` that has a `name`. Otherwise if we do not want to have extra fields for a `Tag` we should probably just return an array of strings?

Reply:
We do it like this so we can easily add extra keys. I personally prefer name as key over tag, but Jens prefers it the other way around :D

Reply:
What I was expecting yes 😄Maybe something to rediscuss?

Reply:
It doesn't really matter that much, does it?

Reply:
Depends if we want to have a `Tag` model containg a field `tag`

Reply:
poke @stivni @jenssegers @jdrieghe (not sure who else helps take decisions about the api, I guess all opinions are welcome)